cassandra-diagnostics-connector-api/pom.xml

```diff
@@ -4,15 +4,37 @@
     <parent>
         <groupId>io.smartcat</groupId>
         <artifactId>cassandra-diagnostics</artifactId>
-        <version>1.0.0</version>
+        <version>1.0.0-SNAPSHOT</version>
     </parent>
     <artifactId>cassandra-diagnostics-connector-api</artifactId>
 
     <dependencies>
         <dependency>
             <groupId>io.smartcat</groupId>
             <artifactId>cassandra-diagnostics-commons</artifactId>
-            <version>1.0.0</version>
+            <version>1.0.0-SNAPSHOT</version>
         </dependency>
     </dependencies>
+    
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <source.level>1.7</source.level>
+        <code.level>1.7</code.level>
+        <version.plugin.compiler>3.5.1</version.plugin.compiler>
+    </properties>
+      
+    <build>
+        <plugins>
+	        <plugin>
+	            <groupId>org.apache.maven.plugins</groupId>
```

Comment:
The whole cassandra-diagnostics-connector-api module will become deprecated when cassandra-diagnostics-connectorXY modules are moved into the parent project.